apis/k8ssandra/v1alpha1/k8ssandracluster_types.go

```diff
@@ -232,6 +233,14 @@ type CassandraClusterTemplate struct {
 	// api heap.
 	// +optional
 	MgmtAPIHeap *resource.Quantity `json:"mgmtAPIHeap,omitempty"`
+
+	// Internode encryption stores which are used by Cassandra and Stargate.
+	// +optional
+	ServerEncryptionStores *encryption.EncryptionStores `json:"server_encryption_stores,omitempty"`
```

Comment:
I know we use snake case for the properties that map directly to properties in cassandra.yaml; otherwise, the convention is camel case. It'd be nice to keep things consistent.

Reply:
sure thing